index.bs

```diff
@@ -532,6 +532,13 @@ that agent.  Agents should discard any authentication message whose
 `auth-initation-token` is set and does not match the `at` provided by the
 advertising agent.
 
+In the `psk-min-bits-of-entropy` field of the [=auth-capabilities=] messsage,
+agents may specify the minimum bits of entropy it requires for a PSK, in the
+range of 20 to 60 bits inclusive.  Agents must terminate any connection
+attempting to authenticate with a PSK with fewer than the minimum number of bits
+of entropy and must not attempt to authenticate with an agent that advertises a
```

Comment:
How would we upgrade this protocol to support > 60 bit PSKs in the future?

Reply:
If anyone ever requires more 60 bits and older agents can't do that many bits, then they would necessarily be incompatible.  I suppose we could represent that with a new auth-capability and then a new message field, or maybe a new message type key with a new message capability.

Reply:
Ok, should make sure that any such future extension is safe from downgrade attacks (similar to the existing protocol).